Source/Plugin.BLE/Windows/Adapter.cs

```diff
@@ -153,12 +149,11 @@ private void Device_ConnectionStatusChanged(BluetoothLEDevice nativeDevice, obje
         protected override void DisconnectDeviceNative(IDevice device)
         {
             // Windows doesn't support disconnecting, so currently just dispose of the device
-            Trace.Message($"DisconnectDeviceNative from device with ID:  {device.Id.ToHexBleAddress()}");
+            Trace.Message($"DisconnectDeviceNative from device with ID:  {device.Id.ToHexBleAddress()}");            
```

Comment:
... and here. Not a big deal, you might say. But if people later edit this file with an editor that automatically removes this, their commits will touch these lines, which can hinder blaming and git archeology.

Reply:
Sorry for being pedantic here 🙈

Reply:
Its's ok - I have formatted the documents now